src/protocol/document.jl

```diff
@@ -61,6 +61,12 @@ end
     position::Position
 end
 
+@dict_readable struct VersionedTextDocumentPositionParams
+    textDocument::TextDocumentIdentifier
+    version::Union{Int,Nothing}
```

Comment:
Should this really be a `Union`? Wouldn't it make more sense to just require `Int` here all the time?

Reply:
Yes, I just copy-pasted this from the "official" `TextDocumentPositionParams` above.